index.html

```diff
@@ -13,17 +13,45 @@
  <header class="header">
   <nav class="nav">
    <ul class="nav_list">
-    <li class="nav_item"> <a class="nav_link" href="#"> Home </a> </li>
+    <li class="nav_item"> <a class="Home" href="#"> Home </a> </li>
     
-    <li class="nav_item"> <a class="nav_link" href="#"> About </a> </li>
+    <li class="nav_item"> <a class="About" href="#"> About </a> </li>
    
-    <li class="nav_item"> <a class="nav_link" href="#"> Blog </a> </li>
+    <li class="nav_item"> <a class="Blog" href="#"> Blog </a> </li>
 
-    <li class="nav_item"> <a class="nav_link" href="#"> Contact Us </a> </li>
+    <li class="nav_item"> <a class="Contact_Us" href="#"> Contact Us </a> </li>

```

Comment:
Hi @DanielNcarter, I suggest you use the same class name for your <a> tags (e.g. class="nav__link").